js/page-specific/senior-expo-booths.js

```diff
@@ -1,126 +1,140 @@
 namespacer('seniorExpo.pageSpecific');
 
 seniorExpo.pageSpecific.seniorExpoBooths = (($, undefined) => {
-	
-	const 
-		
-		/**
-		 * Load everything up and display the SVG.
-		 */
-		display = (svgFilePath, width, height) => {
-			if (!Snap) {
-				console.log('"Snap" library not loaded.');
-				return;
-			}
 
-			const floorplan = Snap('.floorplan');
-
-			Snap.load(svgFilePath, fragment => {
-				snapLoadHandler(fragment, floorplan, width, height);
-			});			
-		},
-
-		/**
-		 * Load up the SVG, highlight the booths, and attach the click handler
-		 */
-		snapLoadHandler = (fragment, floorplan, width, height) => {
-			const parentElement = floorplan.append(fragment),
-				$parentElement = $(parentElement.node),
-				booths = parentElement.selectAll('svg > g > g');	
-
-			$.ajax('/PowerOfAge/exhibitors/2017-exhibitors.html')
-				.done(boothAssigmentData => {
-					let extractedBoothAssigmentData = extractDataFromHtml(boothAssigmentData);
-
-					highlightAssignedBooths(extractedBoothAssigmentData, booths, (snapElement, boothData) => {
-						snapElement.click((clickEvent) => {
-							svgElementClickHandler(clickEvent, boothData);
-						});
-						snapElement.hover((clickEvent) => {
-							svgElementClickHandler(clickEvent, boothData);
-						});
-					});
-				})
-				.fail(errorResponse => {
-					console.log(errorResponse);
-				});
-		},
+	const $floorplan = $('.floorplan');
 
-		/**
-		 * Extract data from the HTML table 
-		 */
-		extractDataFromHtml = htmlData => {
-			const htmlArray = Array.prototype.slice.call($(htmlData)),
-				data = [];
+	let lastBoothId = '';
 
-			let $tableRows;
+	/**
+	 * Load up the SVG, highlight the booths, and attach the click handler
+	 */
+	const loadHtml = (callback) => {
+		$.ajax('/PowerOfAge/exhibitors/2017-exhibitors.html')
+			.done(boothAssigmentData => {
+				callback(boothAssigmentData);
+			})
+			.fail(errorResponse => {
+				console.log(errorResponse);
+			});
+	};
 
-			for (let i = 0; i < htmlArray.length; i++) {
-				const $target = $(htmlArray[i]).find('#SEContentResults');
+	/**
+	 * Extract data from the HTML table 
+	 */
+	const extractDataFromHtml = htmlData => {
+		const htmlArray = Array.prototype.slice.call($(htmlData)),
+			data = [];
 
-				if ($target.length) {
-					$tableRows = $target.find('tbody tr');
-					break;
-				}
+		let $tableRows;
+
+		for (let i = 0; i < htmlArray.length; i++) {
+			const $target = $(htmlArray[i]).find('#SEContentResults');
+
+			if ($target.length) {
+				$tableRows = $target.find('tbody tr');
+				break;
 			}
+		}
+
+		$.each($tableRows, (index, row) => {
+			const $cols = $(row).find('td'),
```

Comment:
Either use a variable declaration here, or move just take the object and directly pass that into the data.push ?